public/javascripts/search-bar.js

```diff
@@ -0,0 +1,15 @@
+const searchBar = document.forms['search-box'].querySelector('input')
+
+searchBar.addEventListener('keyup', function (e) {
```

Comment:
You'll most likely want to debounce or at least throttle this event to ensure smooth performance over a large dataset (like all cards at once)